app/models/league.rb

```diff
@@ -56,4 +57,16 @@ def round_robin_pairings(teams_array)
   def can_join?(user)
     single? && (not has_participant?(user)) && (not deadline_has_passed?)
   end
+
+  def date_for_gameday(gameday)
+    ((gameday - 1) * gameday_duration).days.since startdate
+  end
+
+  def startdate_for_gameday(gameday)
+    date_for_gameday gameday
```

Comment:
Why should this be neccessary?

Reply:
Just readability and it might need some logic later if you want to be able to have multiple gamedays at the same day (duration minimum is 1 day atm)

Reply:
I would say alright but some would say do not write code until you need it (SWA).

Reply:
I suppose I'll rename date_for_gameday to startdate_for_gameday then ^^